examples/exotica_examples/launch/PythonPlanIKWithJntSmooth.launch

```diff
@@ -0,0 +1,21 @@
+<launch>
+
+  <arg name="debug" default="false" />
+  <arg unless="$(arg debug)" name="launch_prefix" value="" />
+  <arg     if="$(arg debug)" name="launch_prefix" value="xterm -e gdb --args python" />
+
+  <param name="robot_description" textfile="$(find exotica_examples)/resources/robots/lwr_simplified.urdf" />
+  <param name="robot_description_semantic" textfile="$(find exotica_examples)/resources/robots/lwr_simplified.srdf" />
+
+  <!-- joy node -->
+  <node respawn="true" pkg="joy"
+        type="joy_node" name="joy_node" >
```

Comment:
The package should list this as a dependency.

Reply:
Covered in latest commit.

Reply:
Not quite. Please change to ``exec_depend`` rather than full ``depend``.